easybuild/easyconfigs/m/mold/mold-1.7.1-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'mold'
+version = '1.7.1'
+
+homepage = 'https://github.com/rui314/mold'
+description = "mold is a high-performance drop-in replacement for existing Unix linkers."
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://github.com/rui314/mold/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['fa2558664db79a1e20f09162578632fa856b3cde966fbcb23084c352b827dfa9']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+]
+dependencies = [
+    ('zlib', '1.2.12'),
+    ('OpenSSL', '1.1', '', SYSTEM),
+]
+
+configopts = "-DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=c++"
```

Comment:
@SebastianAchilles `-DCMAKE_BUILD_TYPE=Release` is used by default by `CMakeMake`, so that can be removed.

What's up with the `-DCMAKE_CXX_COMPILER=c++`, is that strictly necessary? `CMakeMake` already specifies `CMAKE_CXX_COMPILER` by default too...

Reply:
I was following the instructions in https://github.com/rui314/mold/#compile-mold too much. :wink: 

Update in https://github.com/easybuilders/easybuild-easyconfigs/pull/16787/commits/8e84eb93b72ca85a67339a41e2e4a6ea04ab63d0